### PR TITLE
add bookmarkToggle

### PR DESCRIPTION
**Pull request by magdalenakierat**
Thank you for checking my bookmark toggle!

### CODE REVIEW

File: bookmark.html [hunk 1]
```diff
@@ -14,7 +14,9 @@ <h1>Bookmarks</h1>
     </header>
     <main>
       <article data-js="card">
-        <button class="material-symbols-outlined">bookmark</button>
+        <button data-js="bookmark" class="material-symbols-outlined">
```
Comment (PReichetanz) [inline bookmark.html]: I think you can use BEM naming here as well
```suggestion
        <button data-js="bookmark" class="card__bookmark">
```

File: bookmark.html [hunk 2]
```diff
@@ -37,7 +39,9 @@ <h2>Question</h2>
         </ul>
       </article>
       <article data-js="card">
-        <button class="material-symbols-outlined">bookmark</button>
+        <button data-js="bookmark" class="material-symbols-outlined">
```
Comment (PReichetanz) [inline bookmark.html]:
```suggestion
        <button data-js="bookmark" class="card__bookmark">
```

File: index.html [hunk 3]
```diff
@@ -14,7 +14,12 @@ <h1>Quiz-App</h1>
     </header>
     <main>
       <article data-js="card">
-        <button class="material-symbols-outlined">bookmark</button>
+        <button
+          data-js="bookmark"
+          class="material-symbols-outlined card__bookmark--not-active"
```
Comment (PReichetanz) [inline index.html]:
```suggestion
          class="card__bookmark"
```
If active:
```suggestion
          class="card__bookmark card__bookmark--active"
```

File: index.html [hunk 4]
```diff
@@ -37,7 +42,12 @@ <h2>Question</h2>
         </ul>
       </article>
       <article data-js="card">
-        <button class="material-symbols-outlined not-selected">bookmark</button>
+        <button
+          data-js="bookmark"
+          class="material-symbols-outlined card__bookmark--not-active"
```
Comment (PReichetanz) [inline index.html]: See above.

File: index.html [hunk 5]
```diff
@@ -60,7 +70,12 @@ <h2>Question</h2>
         </ul>
       </article>
       <article data-js="card">
-        <button class="material-symbols-outlined">bookmark</button>
+        <button
+          data-js="bookmark"
+          class="material-symbols-outlined card__bookmark--not-active"
```
Comment (PReichetanz) [inline index.html]: See above.

File: js/bookmark.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+function toggleBookmark() {
+  const bookmarks = document.querySelectorAll("[data-js=bookmark]");
+  bookmarks.forEach((bookmark) => {
+    bookmark.addEventListener("click", () => {
+      bookmark.classList.toggle("card__bookmark--not-active");
```
Comment (PReichetanz) [inline js/bookmark.js]:
```suggestion
      bookmark.classList.toggle("card__bookmark--active");
```

File: style/components/questioncard.css [hunk 7]
```diff
@@ -25,7 +25,7 @@ article .material-symbols-outlined {
   background-color: transparent;
 }
 
-article .material-symbols-outlined.not-selected {
+article .material-symbols-outlined.card__bookmark--not-active {
```
Comment (PReichetanz) [inline style/components/questioncard.css]: I don't think that you need the type selector `article` here; it should be sufficient to use
```suggestion
.card__bookmark--active {
```
and above, you change `article .material-symbols-outlined` into `.card__bookmark` and switch the styles for the **not active bookmark**.

**Review by PReichetanz — COMMENTED** [review]
Nice work! I suggested some ideas about the BEM convention.

File: bookmark.html (hunk 1 above)
Comment (magdalenakierat) [reply bookmark.html]: Unfortunately it seems to be impossible to change the class name "material-symbols-outlined" to anything else without losing the icon font.

File: index.html (hunk 3 above)
Comment (magdalenakierat) [reply index.html]: At least it is possible to change the bookmark-class to "card__bookmark--active" if active. Thanks for the hint! :)

**Pull request merged**